flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/connector/hbase/util/HBaseTimestampGenerator.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.hbase.util;
+
+/** Generate timestamp for HBase mutation. */
+public abstract class HBaseTimestampGenerator {
```

Comment:
Is there a chance we will have another implementation for this class in the future? Even if yes, I think we can introduce the extra abstraction layer when it will be necessary. For now, a simple class, e.g. `StronglyIncreasingTsGenerator` with 1 public method that returns the ts would be enough, resulting a simpler and cleaner solution.

Reply:
> Is there a chance we will have another implementation for this class in the future? Even if yes, I think we can introduce the extra abstraction layer when it will be necessary. For now, a simple class, e.g. `StronglyIncreasingTsGenerator` with 1 public method that returns the ts would be enough, resulting a simpler and cleaner solution.

Thank you for your feedback. I will make some modifications according to your suggesion.